website/content/partials/oidc-jwt-claims.mdx

```diff
@@ -0,0 +1,73 @@
+### Bound claims
+
+Once a JWT has been validated as being properly signed and not expired, the
```

Comment:
What about OIDC? You've included the partial in both of the new docs, but it seems to be written specifically for JWT?

Reply:
This partial is relevant for both OIDC and JWT auth methods. JWTs are a part of the OIDC authentication flow. They are also a part of the JWT authentication flow.